operators/olm-common-terms.adoc

```diff
@@ -0,0 +1,12 @@
+[id="olm-understanding-common-terms"]
+= Operator Framework glossary of common terms
+include::modules/common-attributes.adoc[]
+:context: olm-understanding-olm
+
+toc::[]
+
+This topic provides a glossary of common terms related to the Operator
+Framework, including Operator Lifecycle Manager (OLM) and Operator SDK, for both
```

Comment:
In `modules/osdk-building-bundle-image.adoc`, you write "the Operator SDK" - not sure if you want the article here.